multiqc/modules/hicexplorer/hicexplorer.py

```diff
@@ -51,17 +51,17 @@ def __init__(self):
         keys_categorization_of_reads_considered = ['Pairs mappable, unique and high quality', 'One mate unmapped',
                                                    'One mate not unique', 'One mate low quality']
         keys_mappable_unique_and_high_quality = ['Pairs used', 'Self ligation (removed)', 'Same fragment', 'Self circle',
-                                                'Dangling end', 'One mate not close to rest site', 'Duplicated pairs']
+                                                 'Dangling end', 'One mate not close to rest site', 'Duplicated pairs']
         keys_list_contact_distance = ['Short range', 'Long range', 'Inter chromosomal']
         keys_list_read_orientation = ['Inward pairs', 'Outward pairs', 'Left pairs', 'Right pairs', 'Inter chromosomal']
 
         # prepare the detail report section
         self.add_section(
-            name = 'Mapping statistics',
-            anchor = 'hicexplorer_categorization_of_considered_reads',
-            plot = self.hicexplorer_create_plot(keys_categorization_of_reads_considered, 'HiCExplorer: Categorization of considered reads', 'categorization'),
-            description = 'This shows how the sequenced read pairs were mapped and those filtered due to mapping problems.',
-            helptext = '''
+            name='Mapping statistics',
```

Comment:
Hah, style war! Apologies, I think it was me who went through adding spaces around all of these.

![spaces](https://www.bingeclock.com/memes/silicon-valley___spaces_instead_of_tabs.jpg)

Reply:
I am always using my default automatic linter to lint it for me. My apologies if this is breaking any of your style guides.

Reply:
It's no problem - I'm pretty relaxed when it comes to style (at least I think I am..?).

What's the linting tool out of curiosity?

Reply:
I am using autopep8 in combination with VS code. It's just a key command to let the linting run :)